system/otherLoadTest/playlist.xml

```diff
@@ -35,6 +35,7 @@
 		<testCaseName>ClassLoadingTest</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
+			<variation>-Xgcpolicy:gencon -Xgc:dynamicBreadthFirstScanOrdering</variation>
```

Comment:
Since this jvm option is specific openj9/ibm, please add the variation in test target that specific to openj9/ibm. i.e.,  ClassLoadingTest_special